modules/Dasboard/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+
+import { ActionIcon, Anchor, Button, Group, Menu, Title } from "@mantine/core";
+import { FaPlus } from "react-icons/fa6";
+import { useSelector } from "react-redux";
+
+import { TRootState } from "@/shared/redux/store";
+
+import { useStyles } from "./Navbar.styles";
+
+const Navbar = () => {
+  const user = useSelector((state: TRootState) => state.auth.user);
```

Comment:
This should be typed. You can find something called useAppSelector in the frontend template and use that

Reply:
Additionally, create a selector const in your reducer file to encapsulate the state logic and import it here.

So this import would look something like this -

```const user = useAppSelector(userSelector)```